charts/member-agent/templates/deployment.yaml

```diff
@@ -84,6 +87,11 @@ spec:
           volumeMounts:
           - name: provider-token 
             mountPath: /config
+          {{- if eq .Values.propertyProvider "azure" }}
+          - name: cloud-provider-config
+            mountPath: /etc/kubernetes/provider
+            readOnly: true
+          {{- end }}
         - name: refresh-token
```

Comment:
Around this container with `{{- if not .Values.useCAAuth }}` and `{{- end }}` after line 112.